packages/ui/src/helpers/logger/logger.ts

```diff
@@ -0,0 +1,14 @@
+import { ConsoleLogger } from '@aws-amplify/core/internals/utils';
+
+type LoggerCategory =
+  | 'AccountSettings'
+  | 'Auth'
+  | 'Geo'
+  | 'Icon'
+  | 'Notifications'
+  | 'StateMachine'
+  | 'StorageImage'
+  | 'StorageManager';
+
+export const getLogger = (category: LoggerCategory) =>
```

Comment:
nit: could be `getUILogger` to indicate that it's prefixing with `AmplifyUI:`?
```suggestion
export const getLogger = (category: LoggerCategory) =>
```

Reply:
Is that necessary? It's in the ui package, not seeing a future where we would have different or multiple prefixes.

Reply:
Yeah, it was a loosely held nit. I'm good with getLogger